docs/community/com-query/grafana-connector.md

```diff
@@ -8,23 +8,30 @@ nav_order: 2
 
 # Grafana Connector
 
-Molecula provides a Grafana plugin, which can be used to interact with FeatureBase (4.0 or higher <!-- TODO is this correct? -->) to run queries and view visualizations of results. Note that this is unrelated to using Grafana for operational monitoring of the FeatureBase cluster. For details on that, see the [monitoring reference page](/docs/community/com-monitoring/old-monitoring)
+Featurebase provides a Grafana plugin, which can be used to interact with FeatureBase (4.0 or higher <!-- TODO is this correct? -->) to run queries and view visualizations of results. Note that this is unrelated to using Grafana for operational monitoring of the FeatureBase cluster. For details on that, see the [monitoring reference page](/docs/community/com-monitoring/old-monitoring)
 
 ## Setup
 
 ### Install Grafana
-For the best experience, [install](https://grafana.com/grafana/download) Grafana version 7 using the binary, not brew. The plugin requires version 7 or greater, but version 7 works best. For convenience, here are the commands for mac. For other platforms, check the link.
+For the best experience, [install](https://grafana.com/grafana/download) Grafana version 9 using the binary, not brew. The plugin requires version 7 or greater, but version 7 works best. For convenience, here are the commands for mac. For other platforms, check the link.
 
+For the best experience, [install](https://grafana.com/grafana/download) Grafana using the binary, not brew. The plugin requires version 7 or greater and works best on version 9.
+
+### Mac OS 
 ```
-curl -O https://dl.grafana.com/enterprise/release/grafana-enterprise-7.5.13.darwin-amd64.tar.gz
-tar -zxvf grafana-enterprise-7.5.13.darwin-amd64.tar.gz
+curl -O https://dl.grafana.com/oss/release/grafana-9.4.7.darwin-amd64.tar.gz
+tar -zxvf grafana-9.4.7.darwin-amd64.tar.gz
 ```
 
-### Install Plugin
-As of FeatureBase 4.3, a Grafana plugin archive should be included in your release archive. Unpack and note the location of the `/dist` folder.
-
+### Linux
+```
+wget https://dl.grafana.com/oss/release/grafana-9.4.7.linux-amd64.tar.gz
+tar -zxvf grafana-9.4.7.linux-amd64.tar.gz
+```
 
-In Grafana's configuration file `grafana-7-x.x/conf/defaults.ini` point the `plugins` field to the `/dist` folder.
+### Install Plugin
+Download the latest Featurebase release from [Featurebase](https://releases.molecula.cloud/). Unpack and locate the `/dist` directory. 
+In Grafana's configuration file `grafana-9-x.x/conf/defaults.ini` point the `plugins` field to the `/dist` directory.
```

Comment:
What `/dist` directory?
Who's created the `/dist` directory?
Is it still called `/molecula/`?